site-docker.yml.sample

```diff
@@ -1,6 +1,9 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all
```

Comment:
I should have done that yes, I'm not even following my own advices haha. Stupidly copied `site.yml.sample`.
Thanks @font